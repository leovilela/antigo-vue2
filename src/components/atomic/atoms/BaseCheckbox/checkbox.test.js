import { shallowMount } from "@vue/test-utils";
import Checkbox from "../BaseCheckbox/checkbox";

const factory = () => shallowMount(Checkbox, {
  stubs: ["Base-Input"],
});

describe("Checkbox", () => {
  test("mounts properly", () => {
    const wrapper = factory();
    expect(wrapper.isVueInstance()).toBeTruthy();
  });

  test("renders properly", () => {
    const wrapper = factory();
    expect(wrapper.html()).toMatchSnapshot();
  });
});
