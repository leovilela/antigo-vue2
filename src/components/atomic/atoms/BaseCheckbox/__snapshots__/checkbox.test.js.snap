// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders properly 1`] = `<label class="checkbox"><input type="checkbox" true-value="true" class="base-checkbox" value="false"> <span class="check"></span> <span class="control-label"></span></label>`;
